src/content-handlers/iiif/modules/uv-shared-module/BaseExtension.ts

```diff
@@ -423,15 +423,17 @@ export class BaseExtension<T extends BaseConfig> implements IExtension {
     });
 
     this.extensionHost.subscribe(IIIFEvents.SHOW_TERMS_OF_USE, () => {
-      let terms: string | null = this.helper.getLicense();
+      let terms: string | null = null;
 
-      if (!terms) {
-        const requiredStatement: ILabelValuePair | null =
-          this.helper.getRequiredStatement();
+      const requiredStatement: ILabelValuePair | null =
+        this.helper.getRequiredStatement();
+      if (requiredStatement?.value) {
+        terms = requiredStatement.value;
+      }
 
-        if (requiredStatement && requiredStatement.value) {
-          terms = requiredStatement.value;
-        }
+      if (!terms) {
+        const license = this.helper.getLicense();
```

Comment:
I believe that `getLicense` can potentially return a null value, so we should only construct the link below if we have a non-null license.